release/release_workflow.wdl

```diff
@@ -259,7 +262,7 @@ workflow release_cromwell {
   call versionPrep as wdl4sPrep { input: 
     organization = organization,
     repo = "wdl4s",
-    file = "build.sbt",
+    file = "project/Version.scala",
```

Comment:
Not sure what you're asking.  This is where the release WDL goes to look for the version number, and when wdl4s was subprojected this is where the version number is now stored.